src/app/views/previous-routes/previous-routes.component.html

```diff
@@ -33,6 +33,11 @@
               </div>
               <div class="previous-routes__header-location-line" [style]="'height: ' + getVerticalSpaceBetween(firstCircle, secondCircle) + 'px'"></div>
             </div>
+            <div class="previous-routes__header-fav">
+              <button class="btn btn-secondary" [ngClass]="routeS.isFavRoute(route) ? 'btn btn-warning' : 'btn btn-secondary'" type="button" (click)="toggleFavState($event, route)">
```

Comment:
```
class="btn" [ngClass]="routeS.isFavRoute(route) ? 'btn-warning' : 'btn-secondary'"
```

soll es wahrscheinlich heißen

Reply:
hm, in meinen tests wird die klasse überschrieben und nicht appended, ich schau nochmal